src/mutation_data.cpp

```diff
@@ -513,6 +513,10 @@ void mutation_branch::load( const JsonObject &jo, const std::string & )
         }
     }
 
+    for( const std::string line : jo.get_array( "integrated_armor" ) ) {
```

Comment:
Should this be a copy?

Reply:
Thank you, fixed.

Reply:
Looks like the answer is yes :D
I wasn't quite sure if it would allow you to make it a reference, I should have made that clear.

Reply:
Sorry for the confusion!